lib/rides/pages/ride_detail_page.dart

```diff
@@ -237,7 +237,7 @@ class _RideDetailPageState extends State<RideDetailPage> {
     setState(() {});
   }
 
-  _showRequestDialog() {
+  void _showRequestDialog() {
```

Comment:
why is this nessacery?

Reply:
nvm read your comment

Reply:
Because of the always_declare_return_types linting rule. It requires you to always declare the return types.

If you don't declare the return types, then dart thinks it has `dynamic` type. That could lead to problems (probably not in this specific case, but generally).